recipes/r-bactdating/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set version = '1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bactdating
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  url: https://github.com/xavierdidelot/BactDating/archive/v1.0.tar.gz
+  sha256: add9c391709e4d340d173cfb29d30ec6815d5ce89ff8038cea981260be01b151
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, testthat, covr
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
```

Comment:
Why is git required here? Is this a leftover?

Consider re-creating the recipe via https://github.com/bgruening/conda_r_skeleton_helper 

That remove not required comments and gets the requirements right.

Reply:
Ah, thanks, yeah, it's a leftover, I ran `conda skeleton` manually, I'll retry with the helper.

Reply:
This package isn't on the CRAN, so I'm applying suggestions/changes manually...

Reply:
all checks passed for the latest